doc/api/magic-snippets.md

```diff
@@ -73,14 +72,22 @@ Returns all the snippets saved in the environment
 
 Arguments:
 
+`{snippet_name}` Return a snippet.
+
 `-d`/`--delete` Delete a snippet.
 
 `-D`/`--delete-force` Force delete a snippet. This may be useful if there are other dependent snippets, and you still need to delete this snippet.
 
 `-A`/`--delete-force-all` Force delete a snippet and all dependent snippets.
 
 ```{code-cell} ipython3
+cte = %sqlcmd snippets gentoo
+print(cte)
```

Comment:
This is currently displaying 
```
WITH
SELECT * FROM penguins.csv where species == 'Gentoo'
```

Is it supposed to display `WITH` even when it is not used as subquery as part of another query?

Reply:
> 1\. Is it supposed to display WITH even when it is not used as subquery as part of another query?

I was following the behavior of `%sqlrender`, but if only showing `WITH` when a snippet is used as subquery is easier to understand on the user side, I will modify the behavior.

> 2\. Also, previously we could sqlrender the final query as well, that uses a snippet as subquery. e.g.
SELECT * from gentoo where bill_length < 50... can you add a similar example ?

For this one, when you mentioned `sqlrender` showing the final query, are you referring to something like the following?
<img width="712" alt="Screen Shot 2023-06-26 at 1 20 06 PM" src="https://github.com/ploomber/jupysql/assets/43221691/b6d0e93e-f9e9-46c2-ae1f-69f2c09520e4">

Reply:
@neelasha23: yes, this is a bug we discovered recently. I reported it on https://github.com/ploomber/jupysql/issues/657, @bbeat2782 will work on it.

@bbeat2782: please work on this first, and then tackle the other one